Makefile

```diff
@@ -0,0 +1,21 @@
+PREFIX ?= /usr/local/bin
+
+target := uoyweek
+source := uoyweek.cpp
+
+.DEFAULT_GOAL := $(target)
+
+.PHONY: run install clean
+
+$(target): $(source)
+	$(CXX) -Wall -Wextra -pedantic $< -o $@
+	chmod +x $@
+
+run: $(target)
+	./$(target) $(ARGS)
+
+install: $(target)
+	cp $^ $(PREFIX)
+
+clean:
+	$(RM) $(target)
```

Comment:
RM looks undefined, `rm -f` usually suffices (-f for cleaning when the target doesn't exist)

Reply:
From <https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html>:

> **RM**
> Command to remove a file; default ‘rm -f’.

Reply:
huh, TIL